### PR TITLE
Add minimal file system blob loader

### PR DESCRIPTION
**Pull request by eyurtsev**
This adds a minimal file system blob loader.

If looks good, this PR will be merged and a few additional enhacements will be
made.

### CODE REVIEW

File: langchain/document_loaders/blob_loaders/file_system.py [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+            raise TypeError(f"Expected str or Path, got {type(path)}")
+
+        self.path = _path
+        self.glob = glob
+
+    def yield_blobs(
+        self,
+    ) -> Iterable[Blob]:
+        """Yield blobs that match the requested pattern."""
+        paths = Path(self.path).glob(self.glob)
```
Comment (dev2049) [inline langchain/document_loaders/blob_loaders/file_system.py:62]: nit: already Path by this point

**Comment by dev2049** [comment]
one nit, otherwise looks good to moi!

File: langchain/document_loaders/blob_loaders/file_system.py (hunk 1 above)
Comment (eyurtsev) [reply langchain/document_loaders/blob_loaders/file_system.py:62]: Good catch

File: langchain/document_loaders/blob_loaders/file_system.py [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+        glob: str = "**/[!.]*",
+        suffixes: Optional[Sequence[str]] = None,
+    ) -> None:
+        """Initialize with path to directory and how to glob over it.
+
+        Args:
+            path: Path to directory to load from
+            glob: Glob pattern relative to the specified path
+                  by default set to pick up all non-hidden files
+            suffixes: Provide to keep only files with these suffixes
```
Comment (hwchase17) [inline langchain/document_loaders/blob_loaders/file_system.py:36]: why suffixes different than glob? shouldnt suffixes be includable in glob?

**Review by hwchase17 — APPROVED** [review]
one question, but assuming theres something im missing lgtm

**Comment by eyurtsev** [comment]
@hwchase17 i think glob patterns don't support specification of multiple suffixes at the same time, so added this as convenience, but can remove

**Pull request merged**